util/src/main/java/tc/oc/pgm/util/bukkit/Platform.java

```diff
@@ -26,7 +27,13 @@ public enum Platform {
           ReflectionUtils.getClassFromName("tc.oc.pgm.util.nms.v1_8.NMSHacks1_8"),
       false),
   SPIGOT_1_9("Spigot", "1.9", NMSHacks1_9.class, true),
-  PAPER_1_9("Paper", "1.9", NMSHacks1_9.class, true);
+  PAPER_1_9("Paper", "1.9", NMSHacks1_9.class, true),
+  SPIGOT_1_10("Spigot", "1.10", NMSHacks1_10_12.class, true),
+  PAPER_1_10("Paper", "1.10", NMSHacks1_10_12.class, true),
+  SPIGOT_1_11("Spigot", "1.11", NMSHacks1_10_12.class, true),
+  PAPER_1_11("Paper", "1.11", NMSHacks1_10_12.class, true),
+  SPIGOT_1_12("Spigot", "1.12", NMSHacks1_10_12.class, true),
+  PAPER_1_12("Paper", "1.12", NMSHacks1_10_12.class, true);
```

Comment:
maybe some representation that allows a range?
```
  SPIGOT_1_10_TO_1_12("Spigot", "1.10-1.12", NMSHacks1_10_12.class, true),
  PAPER_1_10_TO_1_12("Paper", "1.10-1.12", NMSHacks1_10_12.class, true);
```

Also regarding the classes above, being referenced via reflection, would a Supplier fix it? ie: `() -> NMSHacks1_8.class`  and used as a `Supplier<Class<? extends NMSHacksPlatform>>` ?

Reply:
Thinking for a range to have it be done via overloading the constructor to have an extra argument ie:
```java
SPIGOT_1_10_TO_1_12("Spigot", "1.10", "1.12", NMSHacks1_10_12.class, true),
```
I'll give using a supplier a try

Reply:
Supplier fixed it